Library/Formula/kapacitor.rb

```diff
@@ -0,0 +1,193 @@
+require "language/go"
+
+class Kapacitor < Formula
+  desc "Open source time series data processor"
+  homepage "https://influxdb.com"
```

Comment:
I'd rather we waited for the dedicated site before we include this or point to the GitHub repo.